research/object_detection/core/preprocessor.py

```diff
@@ -2284,10 +2284,10 @@ def add_black_patch_to_image(image, idx):
     return image
 
 
-# TODO(barretzoph): Put in AutoAugment Paper link when paper is live.
 def autoaugment_image(image, boxes, policy_name='v0'):
   """Apply an autoaugment policy to the image and boxes.
-
+  
+  Cubuk et al., AutoAugment: Learning Augmentation Strategies from Data.
```

Comment:
arxiv link?

Reply:
@pkulzc I noticed that none of the other referenced papers have arxiv links. Should we break tradition for this paper?

Reply:
Please add this link (https://arxiv.org/abs/1805.09501).